LiteCore/Query/QueryParser.cc

```diff
@@ -717,6 +718,8 @@ namespace litecore {
                 }
                 if (title.empty())
                     title = "*";        // for the property ".", i.e. the entire doc
+            } else {
+                _sql << " AS \"" << title << "\"";
```

Comment:
Use `quoteTableName(title)` instead of putting quotes around it. This will handle edge cases like if there are quotes inside `title`.

Reply:
Sure, I wasn't sure if this was limited to only table names.

Reply:
Wait, doesn't `quoteTableName` do the same thing?  I don't see any edge cases handled...

Reply:
Oops, I thought it did, sorry!